## 不要将面板里的配置提示直接复制进去，否则会出现 chr() 错误，可以复制以下内容
````
{
    "server": "0.0.0.0",
    "server_ipv6": "::",
    "server_port": 8388,
    "local_address": "127.0.0.1",
    "local_port": 1080,

    "password": "m",
    "method": "aes-128-ctr",
    "protocol": "auth_sha1_v4_compatible",
    "protocol_param": "",
    "obfs": "tls1.2_ticket_auth_compatible",
    "obfs_param": "",
    "speed_limit_per_con": 0,
    "speed_limit_per_user": 0,

"additional_ports" : {
    "80": {
        "passwd": "pass",
        "method": "aes-128-ctr",
        "protocol": "auth_aes128_md5",
        "protocol_param": "#",
        "obfs": "tls1.2_ticket_auth_compatible",
        "obfs_param": ""
    },
    "443": {
        "passwd": "pass",
        "method": "aes-128-ctr",
        "protocol": "auth_aes128_md5",
        "protocol_param": "#",
        "obfs": "tls1.2_ticket_auth_compatible",
        "obfs_param": ""
    }
},

    "additional_ports_only" : false, // only works under multi-user mode
    "timeout": 120,
    "udp_timeout": 60,
    "dns_ipv6": false,
    "connect_verbose_info": 1,
    "redirect": "",
    "fast_open": false
}
````

## 服务器时间一定得是UTC
````
date 一下，发现系统时间明明是对的，但是确实这种：Tue Nov 21 14:32:02 EST 2017
则100%出现握手失败：
2017-11-21 14:31:56 INFO     obfs_tls.py:290 tls_auth wrong time
2017-11-21 14:31:56 WARNING  common.py:238 unsupported addrtype 69, maybe wrong password or encryption method

将服务器的时区改成UTC，怎么改自己上网搜
Tue Nov 21 14:31:59 UTC 2017

````
